phpunit.xml.dist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.4/phpunit.xsd"
+         colors="true"
+         verbose="true"
+         bootstrap="vendor/autoload.php">
+    <testsuites>
+        <testsuite name="reCAPTCHA Test Suite">
+            <directory>tests/Unit/V1/</directory>
+        </testsuite>
+    </testsuites>
+    <filter>
+        <whitelist>
+            <directory suffix=".php">src/Google/Photos/Library/V1/</directory>
+        </whitelist>
+    </filter>
+    <logging>
+        <log
+            type="coverage-text"
```

Comment:
Formatting nit: shouldn't the first parameter be on the previous line?

Reply:
This was copy/paste from an existing file, I can reformat it later :)

Reply:
resolved in 5b5e9c6